src/coreclr/inc/profilepriv.h

```diff
@@ -13,6 +13,9 @@
 #ifndef _ProfilePriv_h_
 #define _ProfilePriv_h_
 
+#ifndef FEATURE_PERFTRACING
```

Comment:
Everything else is using FEATURE_EVENT_TRACE so I suspect this should do that as well?

Reply:
I think `FEATURE_PERFTRACING` is more accurate for this spot.

The reason why the build fails when `FEATURE_EVENT_TRACE` is turned off is because this definition is missing when source files including this header is being compiled.

https://github.com/dotnet/runtime/blob/9ff850ea5d29f487961d1c773bd495630aa8d2ea/src/coreclr/vm/eventpipeadaptertypes.h#L11

That definition was guarded with `FEATURE_PERFTRACING`, that's why I guard this defintion by `!FEATURE_PERFTRACING`

But practically, guarding this definition by `!FEATURE_EVENT_TRACE` will also work just fine.

Reply:
ok, so everything related to eventpipe seems to be guarded with FEATURE_PERFTRACING while FEATURE_EVENT_TRACE covers a different trace subsystem (pure etw?), not directly related to eventpipe. Then I would assume all code that relates to eventpipe functions or structures would be guarded with FEATURE_PERFTRACING.